quarkus-extension/engine/runtime/src/main/java/org/camunda/bpm/quarkus/engine/extension/impl/CamundaEngineRecorder.java

```diff
@@ -102,6 +103,12 @@ protected void configureCdiEventBridge(QuarkusProcessEngineConfiguration configu
     return new RuntimeValue<>(processEngine);
   }
 
+  public void fireProcessEngineStartEvent() {
+    // the 'fireEvent()' method is deprecated in CDI 2.0,
+    // but our CDI integration still supports CDI 1.0, so we continue to use it
+    Arc.container().beanManager().fireEvent(new ProcessEngineStartEvent());
```

Comment:
My worry was that we might run into problems since our CDI integration uses CDI 1.0. However, Quarkus ArC uses CDI 2.0, so it makes more sense to use the new method.

Reply:
Does our CDI integration really use CDI 1.0? The `engine-cdi` module was indeed designed for CDI 1.0 and is tested against a Weld version supporting CDI 1.0. However, since we test the functionally against Quarkus ArC which inturns supports CDI 2.0, we should be good to go.

Also, I don't see this event emitting related to the CDI integration.